src/styles/_price.scss

```
@@ -0,0 +1,25 @@
+@use './variables' as *;
+@use './mixins' as *;
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  width: $primary-width;
```

width should be 100% so it would adapt if card change it's width
```suggestion
  width: 100%;
```